client/src/views/page-components/account/AccountHistoryModal/AccountHistoryModal.ts

```diff
@@ -0,0 +1,388 @@
+import { $, createDOMWithSelector, removeOthersClassList } from '@src/utils/helper';
+import './AccountHistoryModal.scss';
+import handleEvent from '@src/utils/handleEvent';
+import { categoryList, matchCategoryAndImg } from '@src/static/constants';
+import PayMethods from '@src/views/components/PayMethods/PayMethods';
+import { samplePay } from '@src/dummyData';
+
+const slideOutTime: number = 1300;
+const dateInputMaxLength: number = 8;
+const moneyInputMaxLength: number = 9;
+const moneyInputMinLength: number = 0;
+const alertShowTime: number = 2000;
+
+/**
+    추가적으로 고려할 부분 : 
+    1.수입 지출!
+    2.윤년 고려!
```

Comment:
vsc extension에서 todo highlight 사용해보시죠!!

```suggestion
* TODO: 추가적으로 고려할 부분 :  1.수입 지출! 2.윤년 고려!
```

<img width="624" alt="스크린샷 2021-08-01 오후 8 13 09" src="https://user-images.githubusercontent.com/19240202/127768739-669eec5f-ae60-4af3-8ccd-c8bb10641a73.png">